lisp/ess-tracebug.el

```diff
@@ -1264,8 +1269,14 @@ If in debugging state, mirrors the output into *ess.dbg* buffer."
       (setq string (replace-regexp-in-string prompt-replace-regexp " \n" string nil nil 1))
 
       ;; replace long prompts
-      (when inferior-ess-replace-long+
-        (setq string (replace-regexp-in-string "\\(\\+ \\)\\{4\\}\\(\\+ \\)+" ess-long+replacement string)))
+      (cond ((and (eq inferior-ess-replace-long+ 'strip)
+                  (or (not ess-eval-visibly)
```

Comment:
Wouldn't this `or` condition also make sense for the next branch? In other words can we have 

```el
(when (and inferior-ess-replace-long+
                   (or (not ess-eval-visibly)..))
   (if ...))
```

Reply:
Sure, I'll add it there